pkg/agents/team5/agent.go

```diff
@@ -151,6 +164,8 @@ func (a *CustomAgent5) Run() {
 		a.treatyOverride()
 		if a.Floor() == a.rememberFloor { // if the agent is still on the same floor it can update its satisfaction
```

Comment:
remeberFloor is updated in dayPassed, so these values should always be equal at this point in the run() code